indico/modules/events/timetable/templates/display/_weeks.html

```diff
@@ -44,16 +45,33 @@
                         {{ day | format_date(format='full', timezone=timezone) }}
                     </div>
                     {% set hidden_count = [] %}
+                    {% set short_contrib_count = [] %}
                     {% for start_time, slot_entries in day_entries.viewitems() %}
+                        {% set start_time_minutes = (start_time.strftime('%-H')|int*60 + start_time.strftime('%-M')|int) %}
+                        {% if day_entries.keys()[loop.index] %}
```

Comment:
You use it that often so maybe assign `{% set days = day_entries.keys() %}` and then use that?